.github/scripts/pr_analyzer.py

```
@@ -0,0 +1,88 @@
+                if re.search(r"\bconsole\.log\s*\(", content):
+                    confidence_score -= 5
+                    deduction_reasons.append(
+                        f"(-5) `console.log` statement found in '{file_path}'."
+                    )
+        except FileNotFoundError:
+            # The file might have been deleted in the PR, which is fine.
+            # No points are deducted in this case.
+            pass
+
+# --- Output the results ---
+print(f"Jules Governance Analysis")
```

Ruff (0.14.13)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)